astropy/io/ascii/docs.py

```diff
@@ -50,6 +48,15 @@
         List of names to include in output.
     exclude_names : list
         List of names to exclude from output (applied after ``include_names``)
+    dtype : list
```

Comment:
Would it not be better to have this be a `dict`? For long tables, counting the columns would seem more annoying...

Reply:
True, but see above, the intent is to mirror `Table.__init__`. Counting is not easier there is I create a new table from scratch either.